spark/src/main/java/org/opensearch/sql/spark/asyncquery/model/SparkSubmitParameters.java

```diff
@@ -86,7 +86,8 @@ public Builder clusterName(String clusterName) {
     }
 
     public Builder query(String query) {
-      config.put(FLINT_JOB_QUERY, query);
+      String wrappedQuery = "\"" + query + "\""; // Wrap the query with double quotes
```

Comment:
Does this work with query with double quotes inside?

Reply:
> Does this work with query with double quotes inside?

Yes, as long as the quotation itself is valid